### PR TITLE
Add first task

### PR DESCRIPTION
**Pull request by MeLJes**
Adding first task files to master brunch

### CODE REVIEW

File: .idea/deployment.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (modestfake) [inline .idea/deployment.xml]: Create `.gitignore` and add this folder `.idea`. Commit and push. Then remove it from GitHub.
Or the same and remove the folder from local folder, then commit and push.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>JS basic</title>
+</head>
+<body>
+
+<ul>
+    <li>
+        <a href="task-1/index.html">Task-1</a>
```
Comment (modestfake) [inline index.html]: folder `task-1-1` and name link title the same as it is in Trello board

File: task-1/index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Task-1</title>
+</head>
+<body>
+
+    <a href="../index.html">Back to home</a>
```
Comment (modestfake) [inline task-1/index.html]: `href="/"` should work

File: task-1/index.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+    <!-- Scripts -->
+    <script src="../js/script.js"></script>
```
Comment (modestfake) [inline task-1/index.html]: Place this script inside `task-1-1` folder. Let's keep the structure scoped by tasks. We're not building a regular website, just learning.

File: js/script.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
+var num = 243; //number
```
Comment (modestfake) [inline js/script.js]: Minor one. We'll learn the styling of code later. But put space after `//`

**Review by modestfake — CHANGES_REQUESTED** [review]

**Comment by MeLJes** [comment]
All things done

**Review by modestfake — APPROVED** [review]

**Pull request merged**